lib/bloc/explorer/explorer_bloc.dart

```diff
@@ -375,13 +375,22 @@ class ExplorerBloc extends Bloc<ExplorerEvent, ExplorerState> {
         String transactionId = vtts.transactionHashes[i];
         ValueTransferInfo? vtt = database.walletStorage.getVtt(transactionId);
         if (vtt != null && !account.vttHashes.contains(transactionId)) {
+          /// this vtt.status check for "confirmed" is in the local database
           if (vtt.status != "confirmed") {
-            ValueTransferInfo vtt =
+            /// this vtt is what is returned from the explorer.
+            ValueTransferInfo _vtt =
                 await Locator.instance.get<ApiExplorer>().getVtt(transactionId);
-            account.addVtt(vtt);
-
-            await database.addVtt(vtt);
-            await database.updateAccount(account);
+            /// if a vtt was previously unconfirmed and the status is now
```

Comment:
Is there any way we can locally tag UTXOs as used to allow having more than one transaction pending?